src/js/profile/mobile/widget/GridView.js

```diff
@@ -299,6 +307,7 @@
 
 				self.on("animationend webkitAnimationEnd", animationEndCallback);
 
+				self.on("change", self._shadeCheckbox);
```

Comment:
You can assign this listener to `self` and extend method `handleEvent`

Reply:
I've tried to do it but `handleEvent` function doesn't seem to be called.

Reply:
This is impossible that the method is not calling.

`self.on("change", self);`

and add case to handleEvent:
```
case "change":
           self._shadeCheckbox(event.target);
       break;
```

Reply:
Thanks for the clarification. Done